testing/camunda-process-test-java/src/main/java/io/camunda/process/test/impl/extension/CamundaProcessTestContextImpl.java

```diff
@@ -206,16 +211,39 @@ public void completeJob(final String jobType) {
     completeJob(jobType, Collections.emptyMap());
   }
 
+  @Override
+  public void completeJobWithExampleData(final String jobType) {
+    final CamundaClient client = createClient();
+    final ActivatedJob job = getActivatedJob(jobType);
+
+    final String logPrefix =
+        String.format("Mock: Complete job [jobType: '%s', jobKey: '%s']", jobType, job.getKey());
+    final BpmnExampleDataReader exampleDataReader =
+        new BpmnExampleDataReader(client, awaitBehavior);
+
+    try {
+      final String exampleDataVariables =
+          exampleDataReader.readExampleData(job.getProcessDefinitionKey(), job.getElementId());
+
+      LOGGER.debug("{} with example data {}", logPrefix, exampleDataVariables);
+      client.newCompleteCommand(job).variables(exampleDataVariables).send().join();
+    } catch (final BpmnExampleDataReaderException e) {
+
+      LOGGER.warn("{} without example data due to errors. {}", logPrefix, e.getMessage(), e);
```

Comment:
🔧 Printing the error duplicated the failure message. We should remove the exception from the log message.


```suggestion
      LOGGER.warn("{} without example data due to errors. {}", logPrefix, e.getMessage());
```